client_app/index_bi.html

```diff
@@ -93,11 +93,14 @@
       });
     }
 
-    // Setup Google Analytics
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    if( ! process.env.DISABLE_GA) {
```

Comment:
This is not going to work - as this is an index.html file that is executed on the client browsers and is not in the node environment. Same goes for the other html files.